src/pages/Authentication/SignUp/SignUp.jsx

```diff
@@ -28,6 +28,7 @@ const SignUp = () => {
     try {
       const { data } = await signupService(signupInput);
 
+      localStorage.setItem("isAuth", true);
       localStorage.setItem("token", data.encodedToken);
       setToken(data.encodedToken);
 
```

Comment:
Just a suggestion here, you can move your handler functions to the context. This will save you the trouble of needing to get the token in all the files.